src/nested_pandas/nestedframe/core.py

```diff
@@ -858,6 +858,114 @@ def max(self, exclude_nest: bool = False, numeric_only: bool = False, **kwargs):
         else:
             return pd.concat([base_max] + nested_maxs)
 
+    def describe(self, exclude_nest: bool = False, percentiles=None, include=None, exclude=None):
+        """
+
+        Generate descriptive statistics, including nested columns with prefix to indicate the source.
+
+        Descriptive statistics include those that summarize the central tendency,
+        dispersion and shape of a dataset's distribution, excluding NaN values,
+        similar to the behavior of `pandas.DataFrame.describe()`.
+
+        Nested columns use `pyarrow` data types for efficiency, which are not always
+        directly compatible with pandas' type-based filtering.
+        - pyarrow strings are not viewed as object type.
```

Comment:
Minor Nit: I just gave an approval, but realized by viewing the [readthedocs build](https://nested-pandas--322.org.readthedocs.build/en/322/reference/api/nested_pandas.NestedFrame.describe.html) that the last two sentences are separated properly because you need a newline separating them in the docstring itself. Would be good to fix, but also very very minor